src/main/java/com/exactprosystems/fix/reader/pcapreader/state/StateUtils.java

```diff
@@ -70,7 +72,8 @@ public State loadState(TcpStreamFactory tcpStreamFactory) {
             Map<String, List<ConnectionAddressState>> streamNameConnectionAddressesMap = state.getStreamNameConnectionAddressesMap();
             if (streamNameConnectionAddressesMap != null && !streamNameConnectionAddressesMap.isEmpty()) {
                 if (!Objects.deepEquals(streamNameConnectionAddressesMap, tcpStreamFactory.getStreamNameConnectionAddressesMap())) {
-                    throw new IllegalStateException("Connection addresses from state and connection addresses from configuration are not equal");
+                    logger.warn("Connection addresses from state and connection addresses from configuration are not equal: {}",
```

Comment:
I'm not sure that read-pcap re-inspect all read files to find packets with new IP

Reply:
could you please look at the code block in the PcapReaderRunnable class
```
                    if (pcapFileEntry.getFirstPacketTimestamp() != null && (pcapFileEntry.getFirstPacketTimestamp().isAfter(finalLastFileFirstPacketTime) ||
                            pcapFileEntry.getFilename().equals(finalLastReadFile))) {
                        files.add(pcapFileEntry);
                    } else {
                        parsedFiles.add(pcapFileEntry.getFilename());
                    }
```